### DISCUSSION TITLE
Subscription stitching

### DISCUSSION THREAD

**Discussion by rghayazi** [ChilliCream/hotchocolate#4908] [Q&A] [answered]
Hi,

I am using HotChocolate.Stitching following this
https://github.com/ChilliCream/hotchocolate-examples/tree/master/misc/Stitching/centralized
but subscription is not working. 
Looking at the last version 13.0.0-preview.15, I see that you've added support for subscription stitching. Any examples on how to use this?

Thank you

**Comment by michaelstaib** [comment, marked as answer]
This is a very early preview build and APIs are chaning.

**Reply by michaelstaib** [reply]
With that I mean we do not yet want people to use it outside of the contributors.

**Reply by rghayazi** [reply]
Thank you!
Any ideas about when this could be available?

**Comment by chrissi517** [comment]
Version 13 is released. Is there now support with subscriptions and schema stitching?
